src/cards/split.ts

```diff
@@ -78,21 +79,23 @@ function add(unit: Unit.IUnit, underworld: Underworld, prediction: boolean, quan
       unit.onDeathEvents.push(id);
     }
   });
-  if (modifier.quantity && modifier.quantity >= splitLimit) {
-    return;
-  }
-  if (unit.image) {
-    unit.image.sprite.scale.x *= scaleMultiplier;
-    unit.image.sprite.scale.y *= scaleMultiplier;
+  for (let i = 0; i < quantity; i++) {
+    if (modifier.quantity && modifier.quantity > splitLimit) {
```

Comment:
What if the quantity should bring you _to_ the split limit?  If a unit has no split, and you cast Split 5 it should bring them to split 2, not return and do nothing

Reply:
Nevermind, I just tested and it works fine